File: Web.Admin/ClientApp/src/app/models/dependency/dependency.ts
import { Attribute } from './attribute';

export class Dependency {

  id: number;
  name: string;
  type: string;
  description: string;
  attributes: Attribute[];

  public DependencyLite(id: number,
    name: string,
    type: string,
    description: string,
    attributes: Attribute[]) {
    this.id = id;
    this.name = name;
    this.type = type;
    this.description = description;
    this.attributes = attributes;
  };

}

File: Libraries/Shared/NIST.CVP.Libraries.Shared.ACVPCore.Abstractions/Models/Dependency.cs
using System.Collections.Generic;

namespace NIST.CVP.Libraries.Shared.ACVPCore.Abstractions.Models
{
	public class Dependency
	{
		public long ID { get; set; }
		public string Name { get; set; }
		public string Type { get; set; }
		public string Description { get; set; }
		public List<DependencyAttribute> Attributes { get; set; } = new List<DependencyAttribute>();
	}
}
